mavros_extras/msg/DistanceSensor.msg

```diff
@@ -0,0 +1,14 @@
+# DISTANCE_SENSOR message data
+
+Header header
+
+sensor_msgs/Range range
+
+uint8 type			# see MAV_DISTANCE_SENSOR enum
```

Comment:
Yes you are right. But since it won't be used, I will delete this custom msg. Anyway, it's useful info for future implementation, so thanks :)